CHANGELOG.asciidoc

```diff
@@ -38,6 +38,7 @@ https://github.com/elastic/beats/compare/v6.0.0-rc1...master[Check the HEAD diff
 *Auditbeat*
 
 *Filebeat*
+- Fix default paths for redis 4.0.1 logs on macOS {pull}5173[5173] 
```

Comment:
For whatever reason it seems Filebeat exists twice (Line 40 and 43), but not part of this PR. Should be solved in a changelog cleanup.